src/dbus.c

```diff
@@ -418,7 +418,11 @@ static void on_name_lost(GDBusConnection *connection,
                          const gchar *name,
                          gpointer user_data)
 {
-        fprintf(stderr, "Name Lost. Is Another notification daemon running?\n");
+        if (dbus_conn)
+                fprintf(stderr, "Cannot connect to '/org/freedesktop/Notifications'."
```

Comment:
* Minor nitpick but I'd say `Cannot acquire` or `Cannot register` is better wording.

* For a slight more serious mistake: `/org/freedesktop/Notifications` is the name of the object, the name is `org.freedesktop.Notifications`. Look [here](https://dbus.freedesktop.org/doc/dbus-tutorial.html#interfaces) for more details.

Reply:
> For a slight more serious mistake: /org/freedesktop/Notifications is the name of the object, the name is org.freedesktop.Notifications. Look here for more details.

The slashes are right.  We have to acquire the path (here: name). We do not acquire the interface. We have to acquire the path. If another process is implementing the interface, we don't care. But if another process is acquiring the path, we've got a problem.

Reply:
I proved myself wrong. After reading the code again, I see that these have to be dots. I think, I have to rule out some DBus misconceptions by reading the docs fully.